packages/react/src/__tests__/ReactProfilerDOM-test.internal.js

```diff
@@ -18,10 +18,7 @@ let ReactCache;
 function initEnvForAsyncTesting() {
   // Boilerplate copied from ReactDOMRoot-test
   // TODO pull this into helper method, reduce repetition.
-  const originalDateNow = Date.now;
-  global.Date.now = function() {
-    return originalDateNow();
-  };
+  // TODO remove `requestAnimationFrame` when upgrading to Jest 24 with Lolex
```

Comment:
Using Jest 24, the test passes without faking `global.requestAnimationFrame`. Not sure if it's due to Lolex or just a newer version of JSDOM